banking-api/src/main/java/com/banking/BankAccountClient.java

```diff
@@ -2,28 +2,28 @@
 
 import com.banking.api.BankAccountApi;
 import com.banking.dto.BankAccountDto;
-
-import java.util.List;
 import java.util.UUID;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
 public class BankAccountClient implements BankAccountApi {
```

Comment:
I do not understand why you have this class. What does it mean?

Reply:
It's a rudiment, I'll delete it